pkgs/applications/emulators/rpcs3/99-ds4-controllers.rules

```diff
@@ -0,0 +1,14 @@
+# DualShock 4 over USB
```

Comment:
I have this exact snippet in my `services.udev.extraRules` and can confirm it works.